examples/plot_compare_reduction.py

```diff
@@ -73,3 +90,29 @@
 plt.ylim((0, 1))
 plt.legend(loc='upper left')
 plt.show()
```

Comment:
It will block the execution of the script before the end. The convention is really to have it only at the end.